mindsdb_native/libs/phases/type_deductor/type_deductor.py

```diff
@@ -185,7 +195,7 @@ def get_column_data_type(arg_tup, lmd):
     if known_type_dist:
         max_known_dtype, max_known_dtype_count = max(
             known_type_dist.items(),
-            key=lambda kv: kv[0]
+            key=lambda kv: kv[1]
```

Comment:
Why the index change here? I'm not sure I follow

Reply:
because the max() should be on values, IDK why it used to be keys :/